Resources/Prototypes/_Funkystation/Entities/Objects/Misc/crystals.yml

```
@@ -0,0 +1,105 @@
+- type: entity
+  parent: FoodBase
+  id: AmmoniaCrystal
+  name: ammonia crystal
+  description: A crystal made from ammonia gas. You can see the liquid gases inside.
+  components:
+  - type: Sprite
+    sprite: _Funkystation/Objects/Weapons/Grenades/ammoniacrystal.rsi
+    state: icon
+  - type: FlavorProfile
+    flavors:
+      - eggplant
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)